contracts/Unibright.sol

```diff
@@ -142,28 +142,28 @@ contract UBTSplitter is Context, Ownable {
      * percentage of the total shares and their previous withdrawals. `token` must be the address of an IERC20
      * contract.
      */
-    function release(address revenueAddress) public virtual {
+    function release() public virtual {
         require(
-            shares[revenueAddress] > 0,
-            "revenueAddress has no shares"
+            shares[msg.sender] > 0,
+            "msg.sender has no shares"
```

Comment:
maybe print msg.sender for easier debugging?

Reply:
how we can do this?